### DISCUSSION TITLE
code and dataset

### DISCUSSION THREAD

**Discussion by 11140321** [yazdanimehdi/AttentionSiteDTI#11] [Q&A] [answered]
Hello, I am a student from China, as I am new to some information in this area, so my ability is relatively weak. I downloaded the code and dataset, but I didn’t find human.txt, so I couldn’t download the pdb file. Below is my email address. Can you send me a copy of your complete dataset, thank you very much! E-mail address: 2785575665@qq.com

**Comment by yazdanimehdi** [comment, marked as answer]
Hello,
You can find humanSeqPdb in: https://github.com/prokia/drugVQA/blob/master/data/humanSeqPdb
and human_data.txt in: https://github.com/masashitsubaki/CPI_prediction/blob/master/dataset/human/original/data.txt
you need to rename data.txt to human_data.txt
